bgetech/sdm72-registers.json

```diff
@@ -0,0 +1,185 @@
+            "id": "phasePower",
+            "readSchedule": "update",
+            "registers": [
+                {
+                    "id": "powerPhaseA",
+                    "address": 12,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Power phase L1",
+                    "unit": "W",
+                    "defaultValue": "0",
+                    "access": "RO"
+                },
+                {
+                    "id": "powerPhaseB",
+                    "address": 14,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Power phase L2",
+                    "unit": "W",
+                    "defaultValue": "0",
+                    "access": "RO"
+                },
+                {
+                    "id": "powerPhaseC",
+                    "address": 16,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Power phase L3",
+                    "unit": "W",
+                    "defaultValue": "0",
+                    "access": "RO"
+                }
+            ]
+        },
+        {
+            "id": "frequencyAndTotalEnergy",
+            "readSchedule": "update",
+            "registers": [
+                {
+                    "id": "frequency",
+                    "address": 70,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Frequency",
+                    "unit": "Hz",
+                    "defaultValue": "0",
+                    "access": "RO"
+                },
+                {
+                    "id": "totalEnergyConsumed",
+                    "address": 72,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Total energy consumed",
+                    "unit": "kWh",
+                    "defaultValue": "0",
+                    "access": "RO"
+                },
+                {
+                    "id": "totalEnergyProduced",
+                    "address": 74,
+                    "size": 2,
+                    "type": "float",
+                    "registerType": "inputRegister",
+                    "readSchedule": "update",
+                    "description": "Total energy produced",
+                    "unit": "kWh",
+                    "defaultValue": "0",
+                    "access": "RO"
+                }
```

Comment:
Looks to me that the only difference to the 630 is the phase energy counters. 
We might consider removing them from the 630 class, or write an own update class skipping that one block...the classes are almost identical and somehow redundant.